spec/models/content_migration/course_copy_assignments_spec.rb

```diff
@@ -175,7 +181,11 @@
 
       new_assignment = @copy_to.assignments.where(migration_id: mig_id(@assignment)).first
       attrs.each do |attr|
-        expect(@assignment[attr]).to eq new_assignment[attr]
+        if @assignment[attr].class == Hash
```

Comment:
I was running into an issue with "eq" not working because sometimes the hash had String keys and other times it was ":" keys... this was the only way I could get it to work. Did you not have any issues?